code/game/objects/items/clown_items.dm

```diff
@@ -13,7 +13,7 @@
 /obj/item/soap
 	name = "soap"
 	desc = "A cheap bar of soap. Doesn't smell."
-	gender = PLURAL
+	gender = NEUTER
```

Comment:
Isn't NEUTER the default for `/obj`?

Reply:
well uhh the only var/gender = NEUTER i could find was in datum/sprite_accessory, its probably fine?

Reply:
Gender is a byond internal and it defaults to neuter
```suggestion
```

Reply:
damn I can't commit to this pr